docs/general/logging.md

```diff
@@ -1,11 +1,13 @@
+[Back to Guides](../README.md)
+
 # Logging
 
-If we are using ActiveModel::Serializers on Rails app by default the `Rails.logger` will be used.
+If we are using ActiveModelSerializers on a Rails app by default the `Rails.logger` will be used.
 
 On a non Rails enviroment by default the `ActiveSupport::TaggedLogging` will be
 used.
 
-If we need to customize the logger we can define this in an initializer:
+You may customize the logger we by in an initializer, for example:
```

Comment:
I think it have a typo.
`we by in an`
is not `You may customize the logger by in an initializer, for example`